src/KBS.Controller/Controllers/ValuesController.cs

```diff
@@ -2,44 +2,34 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using KBS.Infrastructure;
 using Microsoft.AspNetCore.Mvc;
 
 namespace KBS.Controller.Controllers
 {
-    [Route("api/[controller]")]
+    [Route("api/testsuite")]
```

Comment:
You should rename the controller and use the original `Route` value here.

Reply:
rename to what